modules/apps/headless/headless-asset-library/headless-asset-library-impl/src/main/java/com/liferay/headless/asset/library/internal/resource/v1_0/AssetLibraryResourceImpl.java

```diff
@@ -512,6 +598,11 @@ private <T, E extends Exception> T _getValue(
 		return value;
 	}
 
+	private boolean _isTrashEnabled(Group group) {
+		return Boolean.parseBoolean(
```

Comment:
As I pointed out in the previous commits, we should use Can you use `_trashHelper.isTrashEnabled(groupId)` here?